src/Base/GeometryPyCXX.cpp

```diff
@@ -407,3 +407,4 @@ void Vector2dPy::init_type()
 }
 
 }  // namespace Base
+// NOLINTEND(cppcoreguidelines*)
```

Comment:
Unmatched - the only NOLINTBEGIN I see in this one is `// NOLINTBEGIN(readability-identifier-length)`, and that was ended on line 381.

Reply:
I copied that from an older version I was working on some time ago...